File: assets/rest/samples/python/DelegatesAPI.getDelegate.get.py
from risesdk import PublicKey, APIError, Client

api = Client('https://wallet.rise.vision/api/')
delegate_pk = PublicKey.fromhex('7067a911f3a4e13facbae9006b52a0c3ac9824bdd9f37168303152ae49dcb1c0')
try:
    delegate = api.delegates.get_delegate(delegate_pk)
    print(delegate.username) # official_pool
except APIError as err:
    print('API error:', err)

File: assets/rest/samples/elixir/DelegatesAPI.getDelegate.get.exs
import RiseElixir

try do
  result =
    RiseElixir.Delegates.get_delegate(%{
      generatorPublicKey: "7f31f1b13f211ada60b61e47778b8d59e7f0ecb56fd6f9ea9200c805e352f6a8"
    })

  IO.puts(result["forged"])
catch
  err -> "API error #{err}"
end
